dmscripts/export_dos_opportunities.py

```diff
@@ -0,0 +1,90 @@
+import csv
+from dmscripts.models.process_rules import remove_username_from_email_address, format_datetime_string_as_date
+
+# This URL is framework agnostic
+PUBLIC_BRIEF_URL = "https://digitalmarketplace.service.gov.uk/digital-outcomes-and-specialists/opportunities/{}"
+
+DOS_OPPORTUNITY_HEADERS = [
+    "ID", "Opportunity", "Link", "Framework", "Category", "Specialist",
+    "Organisation Name", "Buyer Domain", "Location Of The Work",
+    "Published At", "Open For", "Expected Contract Length", "Applications from SMEs",
+    "Applications from Large Organisations", "Total Organisations", "Status", "Winning supplier",
+    "Size of supplier", "Contract amount", "Contract start date"
+]
+
+
+def _build_row(brief, brief_responses):
+    winner = None
+    applications_from_sme_suppliers = 0
+    applications_from_large_suppliers = 0
+
+    for brief_response in brief_responses:
+        if brief_response['supplierOrganisationSize'] == 'large':
+            applications_from_large_suppliers += 1
+        else:
+            applications_from_sme_suppliers += 1
```

Comment:
This is super fussy but could we have a blank line after this one to make it clear the next conditional block is unrelated?